pkg/api/repository_package_groups.go

```diff
@@ -18,6 +18,7 @@ type RepositoryPackageGroupCollectionResponse struct {
 
 type SearchPackageGroupResponse struct {
 	PackageGroupName string         `json:"package_group_name"`            // Package group found
```

Comment:
thank you for adding in the ID here! small thing, do you think we should change this in the API docs to say `Name of package group found` to be more explicit? same for environments

Reply:
sure!